### PR TITLE
Run data provisioning command on db VM before test

### PR DESCRIPTION
**Pull request by yoooou**

### CODE REVIEW

File: Jenkinsfile-omec-test-TC1.groovy [hunk 1]
```diff
@@ -289,6 +289,16 @@ node("${params.buildNode}") {
         sh returnStdout: true, script: """ssh sgx-kms-cdr 'pgrep -x dealer'"""
         sh returnStdout: true, script: """ssh sgx-kms-cdr 'pgrep -x dealer-out'"""
       }
+      stage("c3po-dbn1") {
+        timeout(2) {
+          c3po_dbn1_output = sh returnStdout: true, script: """
+          ssh c3po-dbn1 '
+              /home/c3po/db_docs/data_provisioning_users.sh 208014567891234 1122334455 internet 465B5CE8B199B49FAA5F0A2EE238A6BC 25 10.0.4.60 d4416644f6154936193433dd20a0ace0
```
Comment (thakurajayL) [inline Jenkinsfile-omec-test-TC1.groovy:296]: We need to make sure .ci folder is changed in the ngic-rtc. Its going to cause failures in Jenkins. Which one you will commit first ? this change or ngic change ? 
Overall I am ok with this change provided we don't get into deadlock in terms of Jenkins tests.

Comment (yoooou) [reply Jenkinsfile-omec-test-TC1.groovy:296]: I'll submit the other change to ngic and then merge together if looks good

Comment (thakurajayL) [reply Jenkinsfile-omec-test-TC1.groovy:296]: Changes are good :) you have following challenge
1. We will break master branch Jenkins ?
2. We need to make sure c3po & ngic_rtc changes go together.

**Comment by thakurajayL** [comment]
I am not approver for this repo. May be @pingping-lin  can do it.

File: Jenkinsfile-omec-test-TC1.groovy (hunk 1 above)
Comment (yoooou) [reply Jenkinsfile-omec-test-TC1.groovy:296]: Do you mean it might break tests with ngic master branch or openmme master?

**Pull request merged**